phy/plot/waveforms.py

```diff
@@ -283,16 +318,16 @@ def on_key_press(self, event):
         shift = 'Shift' in event.modifiers
 
         # Box scale.
-        if ctrl and key in self._arrows:
+        if ctrl and key in self._arrows + self._pm:
             coeff = 1.1
             u, v = self.box_scale
             if key == 'Left':
                 self.box_scale = (u / coeff, v)
             elif key == 'Right':
                 self.box_scale = (u * coeff, v)
-            elif key == 'Down':
+            elif key in ('Down', '-'):
                 self.box_scale = (u, v / coeff)
-            elif key == 'Up':
+            elif key in ('Up', '+'):
```

Comment:
not sure if = makes sense. and you always have the numpad
there are so many things that are annoying with azerty because of you english people, i won't make you a gift now :p